help/include/links-intro.md

```diff
@@ -0,0 +1,6 @@
+In Zulip, you can insert a named link using Markdown formatting. In addition, Zulip
+automatically creates links for you when you enter:
+
+- A URL
+- A stream name, or a stream name followed by a topic (see [Link to a message or conversation](/help/link-to-a-message-or-conversation))
```

Comment:
This feels misleading -- it sounds like you just need to enter the stream name, which is not accurate -- there's a special syntax here.

Reply:
How about this?

> - An appropriately formatted stream name, or a stream name followed by a topic
  (see also [Link to a message or
  conversation](/help/link-to-a-message-or-conversation))

Reply:
pushed this version